### PR TITLE
BLOGS-1340 - Use parameters.yaml instead of .env

### PR DESCRIPTION
**Pull request by mattgreenham**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: config/parameters_prod.yaml.dist [hunk 1]
```diff
@@ -0,0 +1,9 @@
+parameters:
```
Comment (votemike) [inline config/parameters_prod.yaml.dist]: What is this file for? Is the config/parameters.yaml.dist not sufficient?

Comment (BjornTwachtmann) [reply config/parameters_prod.yaml.dist]: This is the file for use in production. It contains the $variable_names that are replaced by the bake-script with data from cosmos config. Can't really use the same for both unfortunately.

File: web/index.php [hunk 2]
```diff
@@ -2,28 +2,51 @@
...
-if (!isset($_SERVER['APP_ENV'])) {
-    (new Dotenv())->load(__DIR__.'/../.env');
-}
+// Loading config from Dotenv would go here, but we've purposefully decided to
+// stick with using a parameters.yaml file instead of environment variables
 
-$appEnv = $_SERVER['APP_ENV'] ?? 'prod';
-$debugMode = $_SERVER['APP_DEBUG'] ?? ($appEnv === 'dev');
+$env = $_SERVER['APP_ENV'] ?? 'dev';
+$debug = $_SERVER['APP_DEBUG'] ?? !in_array($env, ['prod', 'prod_int', 'prod_test', 'prod_int_fixture', 'prod_test_fixture']);
```
Comment (BjornTwachtmann) [inline web/index.php]: You can remove the prod_int_fixture and prod_test_fixture here, those are for /programmes fixturing

**Pull request merged**